.circleci/config.yml

```diff
@@ -16,30 +16,30 @@
 version: 2.1
 
 orbs:
-  python: circleci/python@0.3.2
+  python: circleci/python@2.0.3
 
 jobs:
   python_test:
     parameters:
       python_ver:
         type: string
-        default: "3.6"
+        default: "3.11"
     docker:
       - image: circleci/python:<< parameters.python_ver >>
     steps:
       - checkout
-      - python/load-cache:
-          dependency-file: requirements_dev.txt
-          key: depsv3-{{ .Branch }}.{{ arch }}-PY<< parameters.python_ver >>
+      # - python/load-cache:  #  This command is unavailable in the orb
+      #     dependency-file: requirements_dev.txt
+      #     key: depsv3-{{ .Branch }}.{{ arch }}-PY<< parameters.python_ver >>
```

Comment:
See https://github.com/ParallelSSH/ssh-python/commit/97cfd5c36e9b65e03841f7301b22bb56ca764854

iirc this is now automated and doesn't have to happen explicitly

Reply:
That is what I was thinking, so can remove it in follow-up commits.